src/client/main.go

```diff
@@ -561,19 +619,18 @@ func prepareRemoteNvim(nvrhContext *nvrh_context.NvrhContext, nv *nvim.Nvim, ver
 	})
 	nv.RegisterHandler("open-url", RpcHandleOpenUrl)
 
-	batch := nv.NewBatch()
```

Comment:
Wasn't sure how I'd feel about concating all the scripts into a single string, but it means that nvrh state can be more private, and removes the need for multiple calls to execute stuff in the remote, feels pretty good